css/style.css

```diff
@@ -212,93 +212,104 @@ h5{
 	line-height: 27px;
 	margin-bottom: 25px;
 }
-.card .button{
-	background-color:#f9004d;
-	color: white;
-	text-decoration: none;
-	border: 2px solid transparent;
-	font-weight: bold;
-	padding: 9px 22px;
-	border-radius: 30px;
-	transition: .4s; 
+.card .button {
+  background-color: #f9004d;
+  color: white;
+  text-decoration: none;
+  border: 2px solid transparent;
+  font-weight: bold;
+  padding: 9px 22px;
+  border-radius: 30px;
+  transition: 0.4s; 
 }
-.card .button:hover{
-	background-color: transparent;
-	border: 2px solid #f9004d;
-	cursor: pointer;
+
+.card .button:hover {
```

Comment:
You're doing great fixing the linter errors :+1: 
To better improve, try also adding a space before the curly brace.
Keep it up :1st_place_medal: